client/cl_property.lua

```diff
@@ -211,6 +213,8 @@ function Property:EnterShell()
 
     self:GiveMenus()
 
+    currentPropertyId = self.property_id
```

Comment:
nit place this under line 207 :P (just so that the code is in the same order between Enter and Leave shell - makes readability easier)

Reply:
done i think